broadcast/internal/arc/arc_submit_tx.go

```diff
@@ -27,6 +33,14 @@ func (a *ArcClient) SubmitTransaction(ctx context.Context, tx *broadcast.Transac
 		opt(options)
 	}
 
+	if options.RawFormat {
+		if err := convertToEfTransaction(tx); err != nil {
+			return nil, fmt.Errorf("Conversion to EF format failed: %s", err.Error())
+		}
+	} else if options.BeefFormat {
+		// To be implemented
```

Comment:
It's better to throw panic or error here ;-)

Reply:
Right, done :)